engine/jobmaster/dm/runtime/worker_status.go

```diff
@@ -89,17 +95,18 @@ func (w *WorkerStatus) RunAsExpected() bool {
 
 // InitWorkerStatus creates a new worker status and initializes it
 func InitWorkerStatus(taskID string, unit framework.WorkerType, id frameModel.WorkerID) WorkerStatus {
-	workerStatus := NewWorkerStatus(taskID, unit, id, WorkerCreating)
+	workerStatus := NewWorkerStatus(taskID, unit, id, WorkerCreating, 0)
 	workerStatus.createdTime = time.Now()
 	return workerStatus
 }
 
 // NewWorkerStatus creates a new WorkerStatus instance
-func NewWorkerStatus(taskID string, unit framework.WorkerType, id frameModel.WorkerID, stage WorkerStage) WorkerStatus {
+func NewWorkerStatus(taskID string, unit framework.WorkerType, id frameModel.WorkerID, stage WorkerStage, cfgModRevision uint64) WorkerStatus {
```

Comment:
maybe we can move taskStatus.Task, taskStatus.Unit, taskStatus.CfgModRevision into one parameter taskStatus

OK to do it in future